source/includes/_enrichment.md

```diff
@@ -1,5 +1,779 @@
+         "phone":"+1 425-882-8080",
+         "linkedin_uid":"1035",
+         "founded_year":1975,
+         "publicly_traded_symbol":"MSFT",
+         "publicly_traded_exchange":"nasdaq",
+         "logo_url":"https://zenprospect-production.s3.amazonaws.com/uploads/pictures/6232f7976d60b600019c2644/picture",
+         "crunchbase_url":null,
+         "primary_domain":"microsoft.com",
+         "persona_counts":{
+            
+         },
+         "market_cap":"2099.6B",
+         "industry":"computer software",
+         "keywords":[
+            "business software",
+            "developer tools",
+            "home",
+            "educational software",
+            "tablets",
+            "search",
+            "advertising",
+            "servers",
+            "windows operating system",
+            "windows applications",
+            "platforms",
+            "smartphones",
+            "cloud computing",
+            "quantum computing",
+            "future of work",
+            "productivity",
+            "ai",
+            "artificial intelligence",
+            "machine learning",
+            "laptops",
+            "mixed reality",
+            "virtual reality",
+            "gaming",
+            "developers",
+            "it professional",
+            "computers",
+            "electronics",
+            "mobile phones",
+            "shopping"
+         ],
+         "estimated_num_employees":224000,
+         "snippets_loaded":true,
+         "industry_tag_id":"5567cd4e7369643b70010000",
+         "retail_location_count":96,
+         "raw_address":"1 microsoft way, redmond, washington 98052, us",
+         "street_address":"1 Microsoft Way",
+         "city":"Redmond",
+         "state":"Washington",
+         "country":"United States",
+         "postal_code":"98052",
+         "owned_by_organization_id":null,
+         "departmental_head_count":{
+            "sales":10117,
+            "business_development":9413,
+            "engineering":66732,
+            "media_and_commmunication":1612,
+            "finance":3079,
+            "marketing":5327,
+            "operations":3747,
+            "support":12733,
+            "consulting":5822,
+            "product_management":4099,
+            "data_science":1922,
+            "administrative":350,
+            "education":13249,
+            "human_resources":3818,
+            "arts_and_design":2125,
+            "information_technology":3727,
+            "legal":625,
+            "accounting":476,
+            "entrepreneurship":121
+         }
+      }
+   ]
+}
+```
+
+This endpoint enriches organization information in bulk with info such as industry, company size, etc. based on the domain parameter passed in.
+
+
+`POST https://api.apollo.io/api/v1/organizations/bulk_enrich`
+
+### Request Parameters
+
+| Parameter | Description        | Example    |
+| --------- | ------------------ | ---------- |
+| domains    | The company domain | google.com |
+
+
```

Comment:
Please be explicit on limits and number of records that can be enriched in single request